package-lock.json

```diff
@@ -4257,8 +4257,8 @@
             }
         },
         "lodash": {
-            "version": "4.17.20",
-            "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.20.tgz",
+            "version": "4.17.21",
+            "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.21 .tgz",
             "integrity": "sha512-PlhdFcillOINfeV7Ni6oF1TAEayyZBoZ8bcshTHqOYJYlrqzRK5hagpagky5o4HfCzzd1TRkXPMFq6cKk9rGmA==",
```

Comment:
hash needs to change for lockfiles as well
http://github.com/aws-cloudformation/cloudformation-cli-typescript-plugin/pull/52 should already cover this, but tests need to be fixed